packages/wallet-service/src/redis.ts

```diff
@@ -22,6 +22,13 @@ export const closeRedisClient = (
   return quit();
 };
 
+export const ping = (
+  client: redis.RedisClient,
+): Promise<string> => {
```

Comment:
What's the return of this method?

Reply:
'PONG'. See https://redis.io/commands/ping/